solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -624,6 +622,12 @@ contract RandomBeacon is Ownable {
 
         relay.requestEntry(groupId);
 
+        tToken.safeTransferFrom(
```

Comment:
We could pass the tToken to `requestEntry`, but decided to move the transfer of T tokens here (we already have all the operations on T tokens in `RandomBeacon` and not in libraries, so maybe it fits better here).